apollo-router/src/plugin/mod.rs

```diff
@@ -164,13 +165,15 @@ where
     pub(crate) fn new_builder(
         config: T,
         supergraph_sdl: Arc<String>,
-        supergraph_schema: Arc<apollo_compiler::validation::Valid<apollo_compiler::schema::Schema>>,
+        supergraph_schema: Arc<Valid<Schema>>,
+        subgraph_schemas: Arc<HashMap<String, Arc<Valid<Schema>>>>,
```

Comment:
I removed the updates to the public `new` and `try_new` to preserve backward compatibility, but it occurs to me this will also be an API change, as it affects the derived builder code.

Will just wrapping these parameters in an `Option` be sufficient to make that backward compatible?